### DISCUSSION TITLE
getting started with: (android) Jetpack Compose

### DISCUSSION THREAD

**Discussion by vitoller** [mapsforge/mapsforge#1310] [General]
I am writing an android application with [Jetpack Compose](https://developer.android.com/jetpack/compose) .

I would appreciate a small example on how to use mapsforge in this framework.

I was able to display a (mapsforge) __MapView__ by using an [**AndroidView**](https://developer.android.com/reference/kotlin/androidx/compose/ui/viewinterop/package-summary#AndroidView(kotlin.Function1,androidx.compose.ui.Modifier,kotlin.Function1)) container.

However so far I failed in displaying the actual map.
Any help would be appreciated.

**Comment by michapries** [comment]
Hi @vitoller, 
did you get this to work? We're in the process of transitioning our app to Compose and information about how to use mapsforge is sparse. An answer would be much appreciated.

**Comment by vitoller** [comment]
Hi,

so far I do not have mapsforge running.

currently I use OSMDROID https://github.com/osmdroid/osmdroid for map display

For OSMDROID an issue and gist exist on how to get started. - OSMDROID can handle mapsforge as source, but so far I did not get round to work on it.

Issue: https://github.com/osmdroid/osmdroid/issues/1674

Gist: https://gist.github.com/ArnyminerZ/418683e3ef43ccf1268f9f62940441b1

**Reply by michapries** [reply]
Thanks a lot! I will try to provide an update once we get _something_ running.
